jdaviz/core/template_mixin.py

```diff
@@ -3397,7 +3426,8 @@ def is_cube(data):
             return len(data.shape) == 3
 
         def is_flux_cube(data):
-            return data.label == getattr(self.app._jdaviz_helper._loaded_flux_cube, 'label', None)
+            uncert_label = getattr(self.app._jdaviz_helper._loaded_uncert_cube, 'label', None)
+            return is_cube(data) and not_child_layer(data) and data.label != uncert_label
```

Comment:
What if a cube does not have uncert data at all? What if it also has mask data?

Reply:
> What if a cube does not have uncert data at all?

There's no problem, this just _excludes_ the uncertainty cube from the dropdown that has this filter.

> What if it also has mask data?

We can also add a check on units (although its tricky at this level of the code and I don't want to add expense if we don't need it).  I thought we don't currently parse/support mask cubes though?

Reply:
We parse the mask cube, we just don't automatically add it to a viewer. But doesn't mean user cannot manually do it later.

p.s. Seems to be a redundancy between `prep_data_layer_as_dq` and loading MASK separately but that is a tech debt for another day.